packages/gatsby-theme-carbon/src/components/NextPrevious/NextPrevious.js

```diff
@@ -189,7 +187,11 @@ const NextPreviousContainer = props => {
   }
 
   return (
-    <NextPrevious previousItem={getPreviousItem()} nextItem={getNextItem()} />
+    <NextPrevious
+      isHomepage
```

Comment:
You'll need to grab `isHomepage` from the props when we destructure them at line 102

```suggestion
      isHomepage={isHomepage}
```

The part where you pass in just `isHomepage` or `isHomepage={true}` is when you use the NextPrevious component in the `src/templates/Homepage` template